src/games/strategy/engine/data/GameData.java

```diff
@@ -444,59 +429,63 @@ public void addToAttachmentOrderAndValues(
    */
   public RelationshipTypeList getRelationshipTypeList() {
     ensureLockHeld();
-    return m_relationshipTypeList;
+    return relationshipTypeList;
   }
 
   /**
    * @return a tracker which tracks all current relationships that exist between all players.
    */
   public RelationshipTracker getRelationshipTracker() {
     ensureLockHeld();
-    return m_relationships;
+    return relationships;
   }
 
   public Hashtable<String, TerritoryEffect> getTerritoryEffectList() {
-    return m_territoryEffectList;
+    return territoryEffectList;
   }
 
   public BattleRecordsList getBattleRecordsList() {
-    return m_battleRecordsList;
+    return battleRecordsList;
   }
 
   /**
-   * Call this before starting the game, and before the game data has been sent to the clients, in order to make any
-   * final modifications to
-   * the game data.
+   * Call this before starting the game and before the game data has been sent to the clients in order to make any
+   * final modifications to the game data.
    * For example, this method will remove player delegates for players who have been disabled.
    */
   public void doPreGameStartDataModifications(final PlayerListing playerListing) {
     final Set<PlayerID> playersWhoShouldBeRemoved = new HashSet<PlayerID>();
-    for (final PlayerID p : m_playerList.getPlayers()) {
-      if (p.getCanBeDisabled() && !playerListing.getPlayersEnabledListing().get(p.getName())) {
-        p.setIsDisabled(true);
-        playersWhoShouldBeRemoved.add(p);
-      }
-    }
+    final Map<String, Boolean> playersEnabledListing = playerListing.getPlayersEnabledListing();
+    playerList.getPlayers().stream()
+        .filter(p -> (p.getCanBeDisabled() && !playersEnabledListing.get(p.getName())))
+        .forEach(p -> {
+          p.setIsDisabled(true);
+          playersWhoShouldBeRemoved.add(p);
+        });
```

Comment:
Do you think changing to using stream() here made this loop more readable and maintainable? Personally I think I prefer the original but it might just be that I haven't used streams quite as much. Just seems that there is less parenthesis and punctuation in the original for each loop in this case.